ofl/cairoplay/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,8 @@
+<p>
+Cairo Play is a contemporary multilingual family featuring colored numerals and vowel marks.
+<a href="https://gaber.design/">Mohamed Gaber</a> extended the Latin typeface family <a href="https://fonts.google.com/specimen/Titillium+Web">Titillum Web</a> to support the Arabic script, with a design that is based on the Kufi calligraphic style.
+</p>
+<p>
+The Cairo project is led by <a href="https://gaber.design/">Mohamed Gaber</a>, a type designer based in Cairo, Egypt. 
```

Comment:
Slightly redundant with first line and designer profile below

Reply:
Agreed. Thanks.